BLUEPRINT/systems/plasma.py

```diff
@@ -321,7 +321,7 @@ def calc_p_lh(self):
             The L-H transition power [MW]
         """
         a = self.params.R_0 / self.params.A
-        return P_LH(self.params.rho * 1e20, self.params.B_0, a, self.params.R_0)
+        return P_LH(self.params.rho * 1e21, self.params.B_0, a, self.params.R_0)
```

Comment:
Apologies if my maths is wrong, but are we off by a factor of ten here?

We used to have:
```
0.8 . 1e20 . 1e20 m^-3 
  = 8 . 1e39 m^-3
```
we now have:
```
80 . 1e21 . um^-3 = 8 . 1e22 . 1e18 . m^-3
  = 8 . 1e40 m^-3
```

Reply:
Yep it should still be 1e20 to keep the same thing, weird conversion in the first place as we would now be in `1/pm^-3`

Reply:
sorry I think it should be 1e18 I think because I am storing it under a different unit (so I can use a prefix) therefore the unit itself doesnt come into play

Reply:
probably should just do this: `ureg.Quantity(80, "um^-3").to("m^-3").magnitude`. I will add a converter function somewhere

Reply:
in the previous version the units were specified as a string `10**20 / m^3`  which is not the easiest thing to do in pint because it is not a multiple of 3^3 -> 21 (for the prefix) and pint doesnt usually allow a raw number in a unit (although possible if you want to muck about a bit). 

```python
In [1]: import pint

In [2]: ureg = pint.UnitRegistry(fmt_locale="en_GB")

In [3]: (ureg.Quantity(0.8, "m^-3") * 1e20).to("um^-3")
Out[3]: 79.99999999999999 <Unit('1 / micrometer ** 3')>  # floating point but its the same
```
But the actual value that was passed into the above function was `0.8*1e20` which needs to be the same raw number for the function to act the same. As `self.params.rho == 80` in the new version we need to convert the raw number. Therefore `80*1e18 == 0.8*1e20`. The conversion happens to be the same as `ureg.Quantity(80, "um^-3").to("m^-3").magnitude`

Does that make sense or am I rambling :slightly_smiling_face: ?

Reply:
Yeah, that makes sense. The physical value of `rho` hasn't changed, but the value passed to the function is 100 times more, so we need to multiply by 100 times less here